figma-tokens.json

```diff
@@ -407,80 +445,6 @@
           "type": "colors"
         }
       }
-    },
-    "size": {
```

Comment:
These tokens are deleted but still being referenced in the avatar, icon and currency icon component, please cross check on it